### PR TITLE
Function for whitelisting items on death

### PR DESCRIPTION
**Pull request by nzkfc**
**Describe Pull request**
function for config whitelist PR.

**Questions (please complete the following information):**
- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? Yes 3+ weeks of use on 160 pop server
- Does your code fit the style guidelines? yes
- Does your PR fit the contribution guidelines? yes

### CODE REVIEW

File: server/player.lua [hunk 1]
```diff
@@ -456,14 +456,17 @@ function QBCore.Player.CreatePlayer(PlayerData, Offline)
         end
     end
 
-    function self.Functions.ClearInventory()
-        self.PlayerData.items = {}
-
+function self.Functions.ClearInventory()
+        for k, v in pairs(self.PlayerData.items) do
```
Comment (BerkieBb) [inline server/player.lua:460]:
```suggestion
        if table.type(QBConfig.Player.WhitelistItems) ~= 'empty' then
            for k, v in pairs(self.PlayerData.items) do
```

**Comment by BerkieBb** [comment]
I'm going to include this with the inventory update

**Comment by nzkfc** [comment]
@BerkieBb one thing I noticed is if you go to jail and had the whitelisted items on you when going to jail, it duplicates the items on release, not sure if this was covered by the update?

**Comment by BerkieBb** [comment]
You can check what I did here https://github.com/qbcore-framework/qb-inventory/pull/318
It's a little different frok your implementation

**Pull request closed without merging**